telegram/request/_httpxrequest.py

```diff
@@ -99,8 +101,9 @@ def __init__(
         write_timeout: Optional[float] = 5.0,
         connect_timeout: Optional[float] = 5.0,
         pool_timeout: Optional[float] = 1.0,
-        http_version: str = "2",
+        http_version: str = "1.1",
     ):
+        self.http_version = http_version
```

Comment:
```suggestion
        self._http_version = http_version
```

Please make this a property, since this value will not be changable by the user at runtime